specification/schemas/metadata_schema_v0.2.json

```diff
@@ -0,0 +1,49 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "metadata_schema_v0.2.json",
```

Comment:
What's the version scheme we are using here? Why v0.2?

Reply:
Well, there are many possibilities, as starting here with v0.1 and bump it to v0.2 if there is any change. But what about taking the version scheme of the current release (right now it would be v0.3, or maybe v0.4 as master is the futur release)? And only bump a new version if the schema changed, related to its release? There may be holes in the versioning (eg: v0.3 then v0.6). Or we can bump a new version each docker-app release, even if there isn't any change? WDYT @garethr ?

Reply:
I don't think we should bump the version every release, just as we end up with lots of identical things to track. I can see pros and cons to both other approaches. However we could in ttheory rev the spec without reving the client. Another one might be to use [CalVer](http://calver.org/) which separates it from the client version scheme nicely? 

cc. @chris-crone @vdemeester

Reply:
+1 for this. We should only change the spec version when we change the schema. Think of it like the `DOCKER_API_VERSION`.

Reply:
Can we converge on this subject in a follow-up? I think it can potentially block the PR, and we just need to agree on the versioning schema before the next release, right?

Reply:
> +1 for this. We should only change the spec version when we change the schema. 
:+1: for this too, this should probably start at `0.1` for now :angel:

Reply:
Ok I'll change it to 0.1 👍